README.md

```diff
@@ -31,6 +31,70 @@ Running `yarn start` will spawn a local copy of the explorer.
 To speed up the build during development, you can run `yarn dev` which will skip
 the optimization step.
 
+## Nix Development
```

Comment:
I'd move this to a separate markdown file (but still leave the mention and link in the `README`). It's a bit specialized.